src/main/resources/assets/music_control/lang/en_us.json

```diff
@@ -56,7 +56,16 @@
     "music.volume": "Volume: %s%%",
     "music.none": "None",
 
+    "music.category.null": "None",
+    "music.category.all": "All",
+    "music.category.default": "Default",
+    "music.category.disc": "Discs",
+
+    "music.format.category": "Playing Category: %s with Music: %s",
```

Comment:
This is generating way too long messages, we don't have time to read. If we could put this message on two line, this would be perfect, but seems we can't.

I'd rather keep it simple like before.  
For example `"Playing: %s | %s"`, or better reuse the now playing from minecraft. I actually still think before was better, even though I would keep the translated category name.